src/api/types.ts

```diff
@@ -184,3 +185,9 @@ export interface IRunResult {
    */
   support: ISupportCodeLibrary
 }
+
+export interface PickleWithDocument {
```

Comment:
If this is only used internally it shouldn't be added here, because it will be added to the main entry point for `@cucumber/cucumber/api`.